docs/development/design/network-router.md

```diff
@@ -0,0 +1,114 @@
+# Network Router
+
+> Additional details and running code in [PR #981](https://github.com/nexodus-io/nexodus/pull/981)
+
+## Summary
+
+Add the ability for device's running nexd to access non-nexd devices and act as a router for external networks.
+
+## Proposal
+
+There are two supported use cases.
+
+1. [Network Router with SNAT](#network-router-with-snat) Traffic from the Nexd mesh can reach external non-nexd nodes via source NAT without any configuration required.
+2. [Network Router with SNAT disabled](#network-router-with-snat-disabled) Traffic from the Nexd mesh can reach external non-nexd nodes but require a route to be added for return traffic since SNAT is disabled.
```

Comment:
```suggestion
1. [Network Router with SNAT](#network-router-with-snat): Traffic from the Nexd mesh can reach external non-nexd nodes via source NAT without any configuration required.
2. [Network Router with SNAT disabled](#network-router-with-snat-disabled): Traffic from the Nexd mesh can reach external non-nexd nodes but external nodes require a route to be added for return traffic since SNAT is disabled.
```